app/controllers/api/accounts_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Api::AccountsController < ApplicationController
+  def new
+    @user = current_user
+  end
+
+  def create
+    @user = current_user
+    @user_api = UserApi.find_or_create_by(user_id: @user.id)
+    if valid_password? && params[:accept]
```

Comment:
@rdavid1099 Getting _much_ better ✨ 

Can we move this to `update`? Seems more in line with what is happening in `overwrite_user_api_key`.